homeassistant/components/aranet4/config_flow.py

```diff
@@ -0,0 +1,104 @@
+"""Config flow for Aranet4 integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aranet4.client import Aranet4Advertisement, Version as AranetVersion
+from bleak.backends.device import BLEDevice
+from bleak.backends.scanner import AdvertisementData
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth import async_discovered_service_info
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_VERSION = AranetVersion(1, 2, 0)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Aranet4."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Set up a new config flow for Aranet4."""
+        self._discovered_devices: dict[str, tuple[str, Aranet4Advertisement]] = {}
+
```

Comment:
You have `bluetooth` set in the `manifest.json` so you should implement `async_step_bluetooth`

Example:
https://github.com/home-assistant/core/blob/6e19629539d0dfecd7500e84ff067b2f6b856770/homeassistant/components/govee_ble/config_flow.py#L31

Reply:
I had vaguely been hoping to defer digging into that for a future PR, but I've added it now and tested it (both via automatic tests and manually).